app/views/state_file/landing_page/edit.html.erb

```diff
@@ -65,7 +65,7 @@
               <% end %>
             <% else %>
               <div class="white-group spacing-below-60">
-                <%= t(".#{@state_code}.help_text_html", default: t(".help_text_html")) %>
+                <%= t(".#{@state_code}.help_text_html", default: t(".help_text_html"), filing_year: Rails.configuration.statefile_current_tax_year) %>
```

Comment:
very minor nit: it looks like everywhere else this is set to an instance variable? personally i don't have a preference between the two but we should probably stick to one